### PR TITLE
separated tag-relevant less into own file

### PR DESCRIPTION
**Pull request by nhaagen**

### CODE REVIEW

File: src/UI/templates/default/Button/tag.less [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	&.btn-tag-inactive {
+		cursor: default !important;
+	}
+	&.btn-tag-relevance-verylow {
+		background-color: @btn-tag-bg-relevance-verylow;
+		color: @btn-tag-fg-relevance-verylow;
+	}
+	&.btn-tag-relevance-low {
+    background-color: @btn-tag-bg-relevance-low;
+    color: @btn-tag-fg-relevance-low;
```
Comment (klees) [inline src/UI/templates/default/Button/tag.less]: Fix indentation please. Spaces and Tabs are mixed.

**Review by klees — CHANGES_REQUESTED** [review]
Please fix indentation. Thx!

**Comment by klees** [comment]
@Amstutz Any objections to this?

**Comment by Amstutz** [comment]
I am fine with this.

**Pull request merged**